Kudu.Services/FetchHelpers/OneDriveHelper.cs

```diff
@@ -62,7 +62,7 @@ internal ILogger Logger
             ChangeSet changeSet = null;
             string cursor = _settings.GetValue(CursorKey);
             ChangesResult changes = null;
-            using (_tracer.Step("Getting delta changes with cursor: {0} ...", cursor))
+            using (_tracer.Step("Getting delta changes with cursor: {0}...", cursor.Truncate(5)))
```

Comment:
No. Whole thing after _tracer.Step(  gets in the file name. That's what was producing the exception in the first place.